### DISCUSSION TITLE
MQTT Refresh Time

### DISCUSSION THREAD

**Discussion by hasanbahadirkoca** [ESPresense/ESPresense#930] [Q&A]
I see that data is sent to mqtt every 5 seconds.
But in UI page there is new data in 1.1 seconds.
How can I shorten this time?

**Comment by atakanpeykan** [comment]
Gönderen tarafta sensör mü var yoksa herhangi bir software mi paylaşırsan publish tarafı yardımcı olabiliriz.

**Reply by hasanbahadirkoca** [reply]
Gönderen taraf dediğiniz, ESP32 üzerinde ESPresense yazılımı çalışıyor. Bu da Home Assistant üzerinde kurulu MQTT sunucusuna veri aktarıyor. Özel bir ayar olarak sadece IRK verileri ekledim. 
/ui sayfasında her bir saniyede yenileniyor veri ancak MQTT 5 saniyede bir güncelleme alıyor.

**Comment by Terastar-Paperclip** [comment]
Tech Lead here (English reply, hope that's fine — happy to have someone translate if useful).

This is already configurable. The 5s MQTT cadence is `skip_ms` ("Skip reporting if message age is less than this") in the web UI settings — default 5000ms. The 1.1s you see in the local `/ui` page is a separate, internal websocket refresh rate that isn't tied to `skip_ms` and isn't currently user-adjustable.

To shorten the MQTT interval: lower `skip_ms` (down to 0 for "as fast as the scan loop allows"). Just know that's a real tradeoff — lower values mean more MQTT/broker traffic and more radio/CPU time per node, which is why 5s is the default rather than 1.1s.
